packages/tools/viewer-alpha/webpack.config.js

```diff
@@ -0,0 +1,53 @@
+const path = require("path");
+const webpackTools = require("@dev/build-tools").webpackTools;
+
+module.exports = (env) => {
+    env = env || {};
+    const source = env.source || "dev";
+    return {
+        // Define the mode as development
+        mode: "development",

```

Comment:
I assume this is webpack config for dev-only and is never used for production build, right?

Reply:
Currently yes.